platform-common-pdf/src/test/java/com/softicar/platform/common/pdf/ImageToPdfConverterTest.java

```diff
@@ -8,12 +8,10 @@
 
 public class ImageToPdfConverterTest extends Asserts {
 
-	private static final String MULTI_IMAGE_TIF = "multi-image.tif";
-
 	@Test
 	public void testConvertToPdf() throws IOException {
 
-		var pdfBytes = new ImageToPdfConverter(() -> getClass().getResourceAsStream(MULTI_IMAGE_TIF)).convertToPdf();
+		var pdfBytes = new ImageToPdfConverter(PdfTestFiles.TIF_MULTI_IMAGE.getResource()::getResourceAsStream).convertToPdf();
```

Comment:
Unrelated change: Moved this unsafely-referenced resource to `PdfTestFiles`.

Reply:
ok